### ISSUE TITLE
You should choose directory for POJO files, before call RoboPOJOGenerator

### ISSUE THREAD

**Issue by ar-g** [robohorse/RoboPOJOGenerator#51]
I'm getting `You should choose directory for POJO files, before call RoboPOJOGenerator` when using plugin exactly like in instruction in **AndroidStudio 2.3-rc1**

**Comment by robohorse** [comment]
Yes, because i need to know your package. Select directory in src/ and select "new -> Generate POJO from JSON"

**Comment by ar-g** [comment]
@robohorse It works fine when I pick exactly as you said `src` folder but when I choosing `src/debug/java/..' I'm getting that msg. I believe it might be because my app has many flavors and according folders with classes

**Comment by robohorse** [comment]
hm... i will look it. This is first time such problem occurred. It worked with multi flavors projects.

**Comment by robohorse** [comment]
cant reproduce, sorry

**Comment by ar-g** [comment]
@robohorse I found that this issue occurs to me only on  Ubuntu OS, it works fine at OSX

**Comment by robohorse** [comment]
@ar-g hm... that is strange. Linux has same path separators that OSX and i get file path by idea sdk tools. I will test it on Virtual Box with Ubuntu. Thanks for information

**Comment by anubhavsaxena14** [comment]
While selecting target POJO type, I am getting error
![screen shot_pojo generator error](https://user-images.githubusercontent.com/9932037/48358666-41187a80-e6c1-11e8-93ae-0d01b8a82f20.png)

**Comment by XDcat** [comment]
change it to project.
![image](https://user-images.githubusercontent.com/40662406/155658340-8d366479-ca5e-4f94-9b86-dfd158d87367.png)
